bank_statement_parse_camt/account_bank_statement_import.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+"""Add process_camt method to account.bank.statement.import."""
+##############################################################################
+#
+#    Copyright (C) 2013-2015 Therp BV <http://therp.nl>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+import logging
+from openerp import models
+from .camt import CamtParser as Parser
+
+
+_LOGGER = logging.getLogger(__name__)
```

Comment:
It is a conflict between two conventions. The more general is to use all uppercase for constants (of course python has no real constants, but variables considered to be constant), the odoo one, I agree is to use _logger. I accepted your suggestion to use _logger.